src/js/orders.spec.ts

```diff
@@ -1,10 +1,10 @@
 
 import BN from 'bn.js';
-const abi = require('ethereumjs-abi');
+import abi from 'ethereumjs-abi';
 import {utils, Wallet} from 'ethers';
 import {ecsign} from 'ethereumjs-util';
 
-const DOMAIN_SEPARATOR = '0x24a654ed47680d6a76f087ec92b3a0f0fe4c9c82c26bff3bb22dffe0f120c7f0';
+export const DOMAIN_SEPARATOR = '0x24a654ed47680d6a76f087ec92b3a0f0fe4c9c82c26bff3bb22dffe0f120c7f0';
```

Comment:
Was this computed by hand, is this the keccak hash of the contracts domain separator string?

Reply:
> keccak hash 
yeah, it should be.